src/scripts/main.js

```diff
@@ -354,7 +354,27 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach((person) => {
+  const table = document.querySelector('.dashboard');
```

Comment:
The `document.querySelector('.dashboard')` is called inside the loop, which is inefficient. It should be moved outside the loop to avoid querying the DOM multiple times unnecessarily.